docs/rules/no-private-routing-service.md

```diff
@@ -6,6 +6,7 @@ Disallow the use of:
 
 * The private `-routing` service
 * The private `_routerMicrolib` property (when the `catchRouterMicrolib` option is enabled)
+* The private `router:main` property (when the `catchRouterMain` option is enabled)
```

Comment:
Are there any plans to rename this to `_router:main` or something like that, to make it more obvious that it shouldn't be used?

Reply:
No, I think it will just be removed eventually, and until that happens, it's the job of linting to make it clear that it shouldn't be used.

https://github.com/emberjs/rfcs/blob/master/text/0095-router-service.md#actively-discourage-use-of-private-api